ec2/sc-ec2-windows-jumpcloud-v1.0.0.yaml

```diff
@@ -274,5 +274,5 @@ Outputs:
     Value: !Ref 'InstanceRole'
   IAMInstanceProfile:
     Value: !Ref 'InstanceProfile'
-  ConnectionInstructions:
-    Value: "https://sagebionetworks.jira.com/wiki/spaces/IT/pages/996376579/Connect+to+Provisioned+Instances"
+  ConnectionURI:
```

Comment:
This technically *can* work for Windows hosts, but it will be a Powershell session. We also haven't yet set up the correct AccessApprovedCaller tag for Windows... that needs to happen in a script that runs when the instance first boots.

Reply:
ok, will remove for windows.  can add it when, and if, you decide to enable powershell access